packages/bodiless-demo/src/styleguide/Page/StyleGuideTemplate/BurgerMenu/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { flowHoc, replaceWith } from '@bodiless/fclasses';
+import { withNodeKey } from '@bodiless/core';
+import { asStyleGuideTemplateToken, vitalStyleGuideTemplate } from '@bodiless/vital-templates';
+import { FlowContainerClean } from '@bodiless/vital-flowcontainer';
+import withBurgerMenuDemoVariations from './withBurgerMenuDemoVariations';
+
+const DemoFlowContainer = flowHoc(
+  withBurgerMenuDemoVariations,
+  withNodeKey('defaultcontainer'),
+)(FlowContainerClean);
```

Comment:
Instead of using the flow container here, do we want to use the thing I made that takes the same input as flow container, but displays the component selector as a page? https://pramne6ybu-jvaff4bu65vgm.us-2.platformsh.site/styleguide/